subprojects/base-services/src/main/java/org/gradle/util/GUtil.java

```diff
@@ -261,7 +256,7 @@ public static void saveProperties(Properties properties, OutputStream outputStre
 
     public static void savePropertiesNoDateComment(Properties properties, File propertyFile) {
```

Comment:
Is this method part of the public API? If so, we should mention the change in behavior.

In either case I think we can find a better name for it. It's not about skipping the date now: it's about reproducibility of the output.

Reply:
If it's public API, we should add some Javadoc, too.

Reply:
`org.gradle.util` is not public API, even though some people think it is...
It was a default import at some point, but the default import is deprecated currently.

I agree that it is about reproducibility of the output. I would love to move all those methods to `PropertiesUtils` and deprecate the methods in here (since it was a default import once). WDYT?

Reply:
All of this is about masking exceptions. I don't think utility methods should do that. Why not just use use `PropertiesUtils` directly wherever you need it, and maybe add a `PropertiesUtils.store(Properties, File, ...)` overload?

Reply:
...and remove the exception masking noise from here.

Reply:
Done. I deprecated `GUtil.savePropertiesNoDateComment` and added some convenience methods to `PropertiesUtils`.